charts/openshift-etcd-backup/values.yaml

```diff
@@ -0,0 +1,92 @@
+# Default values for openshift-etcd-backup.
```

Comment:
Values in here are all missing [helm-docs](https://github.com/norwoodj/helm-docs) style comments